GRIDS.md

```diff
@@ -0,0 +1,57 @@
+# Grid listing
+
+## Horizontal grids
+
+2-dimensional transformation grids for horizontal datum shifts.
+
+### USA: NAD27 -> NAD83
+
+*Source*: USGS
+*Format*: CTable2
+*License*: Public Domain
+
+* alaska - Alaska
+* conus - Conterminous U.S.
+* hawaii - Hawaai
+* prvi - Puerto Rico, Virgin Is.
+* stgeorge - St. George Is, Alaska
+* stlrnc - St. Lawrence Is., Alaska
+* stpaul - St. Paul Is., Alaska
+
+### USA: NAD83 -> NAD83
+
+*Source*: USGS
+*Format*: CTable2
+*License*: MIT (presumably, since it is bundled with PROJ.4)
+
+Grid data for high precision conversion of geographic coordinates from
+NAD83 to NAD83.
+
+* FL - Florida
+* MD -  Maryland
+* TN - Tennessee
+* WI - Wisconsin
+* WO - Washington, Oregon, N. California
+
+### Canada: NAD27 -> NAD83
+
+*Source*: [Natural Resources Canada](http://www.nrcan.gc.ca/earth-sciences/geomatics/geodetic-reference-systems/18766)
+*Format* NTv1
+*License*: MIT (presumably, since it is bundled with PROJ.4)
```

Comment:
The license is probably not MIT, public domain more probably.

The copyright file in the Debian packages has the following for the datum grids:
>  All those data are public domain, taken and collected from various
> US/Canada, France, Switzerland and New Zealand public domain resources.

When I was researching the origin of the datum grids, I mostly found by now ancient discussions on the proj mailing list. We should probably include those as references and quote the relevant bits because the list archive is unlikely to survive the test of time.

Reply:
> When I was researching the origin of the datum grids, I mostly found by now ancient discussions on the proj mailing list. We should probably include those as references and quote the relevant bits because the list archive is unlikely to survive the test of time.

Did you by any chance save the references to those somewhere? Or are they easy to find in the archive?